config/initializers/omniauth.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :facebook, ENV['facebook_key'], ENV['facebook_secret'],
+  :image_size => 'large'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.